src/_guides/libraries/create-library-packages.md

```diff
@@ -1,13 +1,11 @@
 ---
 title: Creating packages
-description: Learn how to create library packages in Dart.
+description: Learn how to create packages in Dart.
 ---
 
 The Dart ecosystem uses [packages](/guides/packages)
 to share software such as libraries and tools.
-This page tells you how to create a package,
-with a focus on the most common kind of package,
-[_library_ packages](/tools/pub/glossary#library-package).
+This page tells you how to create the most common kind of [package](/tools/pub/glossary#package).
```

Comment:
nit: I would actually guess this is _not_ the most common type of package. There are probably many more apps out there than there are shared packages, and every app has to be a package (technically not true but true enough).

Apps might create multiple packages just for organization but I get the sense that they don't typically do that.